corrcal/test/test_corrcal_fof.py

```diff
@@ -1,67 +1,76 @@
-import numpy
+import numpy as np
+from scipy.optimize import fmin_cg
+
 from corrcal import corrcal
 
-nn=5
-x=numpy.arange(nn)
-xmat=numpy.repeat([x],nn,axis=0)
-ymat=xmat.copy().transpose()
-
-scat=0.01
-
-xmat=xmat+scat*numpy.random.randn(xmat.shape[0],xmat.shape[1])
-ymat=ymat+scat*numpy.random.randn(ymat.shape[0],ymat.shape[1])
-
-xpos=numpy.reshape(xmat,xmat.size)
-ypos=numpy.reshape(ymat,ymat.size)
-antvec=numpy.arange(xpos.size)
-
-xx=numpy.repeat([xpos],xmat.size,axis=0)
-yy=numpy.repeat([ypos],ymat.size,axis=0).transpose()
-
-antmat=numpy.repeat([antvec],antvec.size,axis=0)
-ant1=antmat.copy()
-ant2=antmat.copy().transpose()
-
-umat=xx-xx.transpose()
-vmat=yy-yy.transpose()
-isok=numpy.where(ant2>ant1)
-
-ant1_org=ant1[isok]
-ant2_org=ant2[isok]
-u_org=umat[isok]
-v_org=vmat[isok]
-vis_org=numpy.random.randn(ant1_org.size)+numpy.complex(0,1)*numpy.random.randn(ant1_org.size)
-
-noise_org=numpy.ones(u_org.size)
-
-vis,u,v,noise,ant1,ant2,edges= corrcal.grid_data(vis_org, u_org, v_org, noise_org, ant1_org, ant2_org)
```

Comment:
Traceback (most recent call last):
  File "test_corrcal_fof.py", line 37, in <module>
    vis,u,v,noise,ant1,ant2,edges= corrcal.grid_data(vis_org, u_org, v_org, noise_org, ant1_org, ant2_org)
ValueError: too many values to unpack (expected 7)

Reply:
Should be fixed in https://github.com/UKZN-Astronomy/corrcal/pull/21/commits/4c92827a0361dfc86daeb5e681e656fba8202a0a.

Reply:
Issue is indeed fixed